smart_contracts/contracts/client/redelegate/src/main.rs

```diff
@@ -0,0 +1,37 @@
+#![no_std]
```

Comment:
Just wondering, is this contract being used anywhere right now? I.e. to validate that it indeed does what it's expected to?

Reply:
Yes, it is being used in the tests, under the auction/bid.rs